websites/M/myCANAL/presence.ts

```diff
@@ -6,62 +6,110 @@ const presence = new Presence({
 		pause: "general.paused",
 		live: "general.live",
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
-let title;
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	containsTerm = (term: string) => {
+		if (document.location.pathname.includes(term))
+			return document.location.pathname;
+	};
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
 			largeImageKey: "logo",
```

Comment:
Imgur links are preferred over Discord asset names.
```suggestion
			largeImageKey: Assets.Logo,
```